integration_test/go_java_launcher_integration_test.go

```diff
@@ -144,7 +144,7 @@ func TestMainMethodWithoutCustomConfig(t *testing.T) {
 }
 
 func TestMainMethodContainerWithoutCustomConfig(t *testing.T) {
-	output := testContainerSupportEnabled(t, "foo", "-XX\\:InitialRAMPercentage=75.0 -XX\\:MaxRAMPercentage=75.0 -XX\\:ActiveProcessorCount=2", []string{})
+	output := testContainerSupportEnabled(t, "foo", "", []string{"-Xmx", "-Xms"})
```

Comment:
If there is a no custom config, or there is a failure to read the custom config, we use the new default behavior, hence this change.